netket/models/mps.py

```diff
@@ -155,4 +161,4 @@ def multiply_tensors(left_tensor, right_tensor):
             return jnp.trace(edge)
 
         # trace the matrix multiplication
-        return jax.vmap(trace_mps)(selected_tensors, edges)
+        return jnp.log(jax.vmap(trace_mps)(selected_tensors, edges))
```

Comment:
Yes this should be enough to solve the issue, @tvieijra all good?

Reply:
yes, this looks perfect!